### ISSUE TITLE
Cross-Platform compatibility ???

### ISSUE THREAD

**Issue by macmeister1967** [jsm174/vpx-standalone-scripts#346]
Hi There. Great work so far.

Running Ubuntu 24.04.1 here... Just wondering if I put these patched scripts in with my SHARED table collection, **will it interfere with my Windows/MacOS builds of regular (non-standalone) VPinballX ??**

NOTE: I am using Wineskin to emulate PC version under macOS, but will be switching to Standalone immediately after I had such success with Linux.

Many thanks.

**Comment by francisdb** [comment]
Yes, these sidecar scripts will also be picked up by the windows version.
However most patched scripts will not break the table on Windows, they just avoid using some specific vbscript constructs. Some scripts make changes regarding filesystem file loading or listing, those might cause issues on windows.

**Comment by macmeister1967** [comment]
Many thanks for the quick reply (in Australia here - 4:46am). There are probably only a dozen scripts I may need, and I guess if I back them up first, I should be fine... NOTE: I will almost certainly be playing under Linux mostly, as the performance and latency is amazing !!!

**Comment by francisdb** [comment]
@jsm174 I think this can be closed?
